src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (invertedRobot.hasOwnProperty(robot[key])) {
+      return null;
+    }
+
+    invertedRobot[`${robot[key]}`] = key;
```

Comment:
The template literal is unnecessary here. You can directly use `robot[key]` as the property name for `invertedRobot` without converting it to a string, as object keys are automatically converted to strings if they are not symbols.